jsk_2016_01_baxter_apc/node_scripts/rbo_segmentation_in_bin_node.py

```diff
@@ -82,8 +82,10 @@ def _callback(self, cloud, img_msg, camera_info):
         # generate a binary image
         self.segmentation()
         try:
-            self.img_pub.publish(self.bridge.cv2_to_imgmsg(
-                    self.predicted_segment, encoding="passthrough"))
+            mask_msg = self.bridge.cv2_to_imgmsg(self.predicted_segment, encoding="passthrough")
+            mask_msg.header.stamp = rospy.Time.now()
```

Comment:
i think stamp means when the mask image is generated.
do you know the public definition?